manifests/init.pp

```diff
@@ -28,7 +29,8 @@
   }
 
   class { "rpmbuilder::packages::essential":
-    epel  => $add_epel,
+    epel         => $add_epel,
+    mock_version => $mock_version,
```

Comment:
I'd rather be safe than sorry. Can we just leave the redundancy in and call it good enough?